tosca-samples/org/ystia/yorc/samples/job/noscheduler/components/scripts/submit.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+echo "Submitting job..."
+export TOSCA_JOB_ID="Job-${RANDOM}"
+
+JOB_TIMESTAMP=`/bin/date +%s%N`
```

Comment:
`$(...)` should be prefered over `` `...` ``

Reply:
ok will change